__tests__/app.test.js

```diff
@@ -71,4 +71,39 @@ describe("api/articles/:article_id", () => {
             expect(body.msg).toBe("article not found!")
         })
     })
+})
+describe("/api/articles/:article_id/comments", () => {
+    test(":) GET 200 - returns all the comments given article id", () => {
+        return request(app).get("/api/articles/3/comments").expect(200).then(({body}) => {
+            const comments = [{
+                "comment_id" : 10,
+                "votes" : 0,
+                "created_at" : expect.any(String),
+                "author": "icellusedkars", 
+                "body": "git push origin master"
+            },{
+                "comment_id" : 11,
+                "votes" : 0,
+                "created_at" : expect.any(String),
+                "author": "icellusedkars", 
+                "body": "Ambidextrous marsupial"
+            }]
+            expect(body).toEqual({comments})
```

Comment:
Add an expect in here to check that the comments come back in the correct order (newest first).